DuckDuckGo/NetworkProtectionTunnelController.swift

```diff
@@ -119,9 +122,38 @@ final class NetworkProtectionTunnelController: TunnelController, TunnelSessionPr
         }
     }
 
-    init(accountManager: AccountManager, tokenStore: NetworkProtectionKeychainTokenStore, persistentPixel: PersistentPixelFiring) {
-        self.tokenStore = tokenStore
+    // MARK: - Enforce Routes
+
+    private var enforceRoutes: Bool {
+        guard featureFlagger.isFeatureOn(.networkProtectionEnforceRoutes) else {
+            return false
+        }
+
+        /// Even though there's a remote feature flag, we still want to allow internal users to disable enforceRoutes
+        /// manually in case they have trouble.  To achieve this we force the setting to ON once, but otherwise
+        /// allow the internal user to disable it again.
+        if !settings.enforceRoutesForceEnabledOnce {
+            settings.enforceRoutesForceEnabledOnce = true
+            settings.excludeLocalNetworks = true
+            settings.enforceRoutes = true
+        }
```

Comment:
The first time the VPN is started with the feature flag ON, we'll enable enforce routes.  But the setting can be overridden by internal users in case there's trouble.